packages/bento-design-system/src/Popover/Popover.tsx

```diff
@@ -29,50 +29,48 @@ type Props = {
  * A Popover element that is rendered in a portal and placed relative to its trigger element using a placement strategy
  * that accounts for edge cases such as not enough space to render the popover in the preferred position.
  */
-export function Popover({
-  children,
-  onClose,
-  triggerRef,
-  placement = "bottom start",
-  offset = 4,
-}: Props) {
-  const overlayRef = useRef(null);
-  const { overlayProps } = useOverlay(
-    {
-      isOpen: true,
-      isDismissable: true,
-      onClose,
-    },
-    overlayRef
-  );
+export const Popover = forwardRef<HTMLElement, Props>(
+  ({ children, onClose, triggerRef, placement = "bottom start", offset = 4 }, ref) => {
+    const overlayRef = useRef<HTMLElement>(null);
+    useImperativeHandle(ref, () => overlayRef.current as HTMLElement);
```

Comment:
why is the cast necessary? `overlayRef.current` should be `HTMLElement`

Reply:
Because of the definition at line 34, `overlayRef.current` is actually `HTMLElement | null`.